extensions-core/avro-extensions/src/main/java/io/druid/data/input/avro/GenericRecordAsMap.java

```diff
@@ -102,7 +102,7 @@ public Object get(Object key)
   {
     Object field = record.get(key.toString());
     if (fromPigAvroStorage && field instanceof GenericData.Array) {
-      return Lists.transform((List) field, PIG_AVRO_STORAGE_ARRAY_TO_STRING_INCLUDING_NULL);
+      return Lists.transform((List) field, PIG_AVRO_STORAGE_ARRAY_TO_STRING_INCLUDING_NULL::apply);
```

Comment:
`PIG_AVRO_STORAGE_ARRAY_TO_STRING_INCLUDING_NULL` is already a function. By adding `::apply`, you transform it twice, but it end up being the same Guava's Function. It doesn't make anything more functional than it is currently